README.md

```diff
@@ -47,11 +41,14 @@ There are several packages that makeup the Dojo 2 platform:
 
 We also have packages that are still in their planning stages and as such might not be part of the initial Dojo 2 release.
 
+* [`dojo/actions`](https://github.com/dojo/actions) - A command like library for Dojo 2 applicationsz
+* [`dojo/app`](https://github.com/dojo/app) - An application framework for Dojo 2
 * [`dojo/crypto`](https://github.com/dojo/crypto) - A set of libraries to deal with cryptography
+* [`dojo/dataviz`](https://github.com/dojo/dataviz) - A data vizualization library for Dojo 2
 
+We have some deprecated packages:
 
-We have deprecated a package:
-
+* [`dojo/dom`](https://github.com/dojo/dom) - A set of APIs for manipulating the DOM
```

Comment:
is this deprecated... I was thinking it might still be useful for someone migrating from 1.x to 2? Maybe not.